components/apimgt/org.wso2.carbon.apimgt.gateway/src/main/java/org/wso2/carbon/apimgt/gateway/handlers/graphQL/GraphQLAPIHandler.java

```diff
@@ -159,6 +173,34 @@ public boolean handleRequest(MessageContext messageContext) {
         return false;
     }
 
+    private Map<String, Object> convertArrayToMap(ArrayList<OMElement> variablesArray) {
```

Comment:
Use void and construct what you need

Reply:
Fixed via https://github.com/wso2/carbon-apimgt/pull/12441/commits/226c49a261240dc5305f96d69aadb6f86edfae84